packages/flame/test/sprite_animation_ticker_test.dart

```diff
@@ -162,5 +162,25 @@ void main() {
       animationTicker.completed;
       expect(animationTicker.completeCompleter!.isCompleted, false);
     });
+
+    test('paused pauses ticket', () async {
+      final sprite = MockSprite();
+      final animationTicker = SpriteAnimation.spriteList(
+        [sprite, sprite],
+        stepTime: 1,
+        loop: false,
+      ).createTicker();
+
+      expect(animationTicker.isPaused, false);
+      expect(animationTicker.currentIndex, 0);
+      animationTicker.update(1);
+      expect(animationTicker.currentIndex, 1);
+      animationTicker.paused = true;
```

Comment:
Maybe do a `animationTicker.update(1)` after pausing to ensure that the `currentIndex` does not change.

Reply:
Did I not do that?!?!  That was the whole point, lol

Reply:
Added and tests pass.